packages/js/client/src/Web3ApiClient.ts

```diff
@@ -419,6 +435,60 @@ export class Web3ApiClient implements Client {
     return subscription;
   }
 
+  @Tracer.traceMethod("Web3ApiClient: resolveUri")
```

Comment:
Why this haven't been added to contextualizeClient?

Reply:
Will add, nice catch